lte/gateway/python/integ_tests/s1aptests/test_sctp_shutdown_while_mme_is_stopped.py

```diff
@@ -55,10 +55,9 @@ def test_sctp_shutdown_while_mme_is_stopped(self):
         self._s1ap_wrapper._s1_util.receive_emm_info()
 
         print("Stopping MME, mobilityd, pipelined, and sessiond service")
-        self._s1ap_wrapper.magmad_util.disable_service('mme')
-        self._s1ap_wrapper.magmad_util.disable_service('mobilityd')
-        self._s1ap_wrapper.magmad_util.disable_service('pipelined')
-        self._s1ap_wrapper.magmad_util.disable_service('sessiond')
+        self._s1ap_wrapper.magmad_util.disable_services(
+            ['mme', 'mobilityd', 'pipelined', 'sessiond'],
```

Comment:
Is it really necessary to still write all 4 services here? The inter-dependencies should now ensure that disabling/enabling one of these does the same for the others anyway (but maybe it is reasonable to keep it for better understanding of what the test wants to do?)

Reply:
That is correct. These is no need for explicitly stopping all of these services. However, enabling one service does not start the others, so we explicitly have to start them again. Instead of just disabling one and enabling four, I decided to explicitly state all services for better readability.